common/OpTestHMC.py

```diff
@@ -55,6 +55,42 @@
 BOOTTIME = 500
 STALLTIME = 5
 
+class OpSecureBootUtilities():
+    '''
+    This class addresses the secureboot related pre-requisites.
+    '''
+
+    def __init__(self, console=None, distro=None):
+        self.console = console
+        self.distro = distro
+
+    def check_kernel_signature(self):
```

Comment:
@rohan-r-deshpande Please add the docstring for more clarity of the method

Reply:
> @rohan-r-deshpande Please add the docstring for more clarity of the method

Done